### DISCUSSION TITLE
Welcome to Job_Helper Discussions!

### DISCUSSION THREAD

**Discussion by Priyansh-15** [Priyansh-15/Job_Helper#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Priyansh-15** [comment]
@koustubh1317 Please Update Readme for Cisco accordingly .
Thanks @Priyansh-15

**Reply by koustubh1317** [reply]
![image](https://user-images.githubusercontent.com/54525819/223172231-bbbadc3f-70c2-4e39-af38-c94e13dadfba.png)
Some error with the first check.
Rest amazon added and readme updated

**Reply by Priyansh-15** [reply]
Will look into it !!

**Comment by Priyansh-15** [comment]
@koustubh1317  Update the website and snips for Google part in Readme.md .

**Comment by Priyansh-15** [comment]
Any updates on the UI Config and Design @koustubh1317 ?

**Reply by koustubh1317** [reply]
Will starting the work on it from today

**Comment by Priyansh-15** [comment]
**_Any updates on the specific tasks ?_** @koustubh1317 @sharma-harshit59 
Need individual tasks done for the integration of each into `main_file.py` .

**Reply by koustubh1317** [reply]
2nd and 3rd pages are complete with mobile view.
4th page is in-progress

**Reply by Priyansh-15** [reply]
sync them with main-stream with changes marked ASAP.

**Comment by koustubh1317** [comment]
Rating (1-10) of skill from description of every job openings @Priyansh-15

**Reply by Priyansh-15** [reply]
Will push the required code for it by tomorrow !!

**Comment by koustubh1317** [comment]
For Setup of UI Part
Node Installations and react setup, follow these commands

Go to following link:-
https://nodejs.org/en/download

Download nodeJs Windows installer(lts)
After download open the installer and leave default setting click on Install

open command prompt in ui directory and run these commands
~ node -v
~ npm -v

Should give you your version for both for them 
Now run
~ npm install

now to run react app 
~ npm start
